config/default.yml

```diff
@@ -3196,6 +3196,17 @@ Style/EndBlock:
   VersionAdded: '0.9'
   VersionChanged: '0.81'
 
+Style/EndlessMethod:
+  Description: 'Avoid the use of multi-lined endless method definitions.'
+  StyleGuide: '#no-endless-methods'
```

Comment:
Btw, we'll also have to update the guideline and its anchor. :-)

Reply:
Should I change it to `#endless-methods` or remove it?

Reply:
I'd go with option 1, as I assume we'll update your guideline proposal and merge it.

Reply:
✅ done